src/main/java/org/example/pages/MainFunctional.java

```diff
@@ -53,10 +54,27 @@ public void hoverElement(By locator) {
 
 	public Boolean isDisplayed(By locator) {
 		try {
+			WebDriverWait wait = new WebDriverWait(driver, 5);
```

Comment:
You use the WebDriverWait in both methods, make sense to move the object initialization to the constructor, in this case you will have reusable object and will save some memory.

Reply:
WebDriverWait wait variable Initialized for class and added to constructor